auth.py

```diff
@@ -0,0 +1,43 @@
+from fastapi.security import OAuth2PasswordBearer
+from fastapi import HTTPException
+from datetime import datetime, timedelta
+import jwt
+
+fake_users_db = {
+  "yamakenji": {
+      "id": "base64id",
+      "username": "yamakenji",
+      "full_name": "Kenji Yamashita",
+      "email": "sample@example.com",
+      "password": "fakehashedsecret",
+      "disabled": False,
+  }
+}
+# for instance 
+secret_key = '09d25e094faa6ca2556c818166b7a9563b93f7099f6f0f4caa6cf63b88e8d3e7'
+
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="/login")
+
+def create_tokens(user_id: str):
+  payload = {
+    'token_type': 'access_token',
+    'exp': datetime.utcnow() + timedelta(minutes=2),
```

Comment:
[REQUEST CHANGES]

Expiration date definitions had better hard-coded. For example, there is a method to read the period from the environment variable.

Reply:
May also need to define a time zone.